src/app/client/src/app/modules/shared/components/offline-application-download/offline-application-download.component.html

```diff
@@ -41,8 +41,8 @@ <h2 class="offline-download-card-heading">
               [telemetryInteractEdata]="{id:'offlineAppDownload-button' , type:'click' , pageid:'offline-application-download' }"
               type="button" class="w-100 d-flex flex-ai-center py-0 sb-btn sb-btn-primary sb-btn-normal mt-8"
               (click)="downloadApp(recomanded_download)">
-              <span class="mr-8" *ngIf="recomandedOS!=='linux'"><img src="assets/images/windows.svg" width="16px"></span>
-              <span class="mr-8" *ngIf="recomandedOS==='linux'"><img src="assets/images/ubuntu.svg" width="16px"></span>
+              <span class="mr-8" *ngIf="recomandedOS!=='linux'"><img src="assets/images/window.svg" width="16px"></span>
```

Comment:
add alt, title tags and rename the window.svg to windows-white.svg (check are they using anywhere else)